runtime/compiler/build/toolcfg/gnu/common.mk

```diff
@@ -288,7 +288,8 @@ ifeq ($(HOST_ARCH),x)
     NASM_INCLUDES=\
         $(J9SRC)/oti \
         $(J9SRC)/compiler \
-        $(J9SRC)/compiler/x/runtime
+        $(J9SRC)/compiler/x/runtime \
+        $(J9SRC)/compiler/microjit/assembly
```

Comment:
Why is this not conditional on `J9VM_OPT_MICROJIT`?

Reply:
Thank you for catching this, it has been moved into a guarded area.